examples/ibm-cis/README.md

```diff
@@ -21,6 +21,7 @@ These types of resources are supported:
 * [ CIS Custom Page](https://cloud.ibm.com/docs/terraform?topic=terraform-cis-resources#cis-custom-page)
 * [ CIS Page Rule](https://cloud.ibm.com/docs/terraform?topic=terraform-cis-resources#cis-page-rule)
 * [ CIS WAF Package ](https://cloud.ibm.com/docs/terraform?topic=terraform-cis-resources#cis-waf-package)
+* [ CIS WAF Rule](https://cloud.ibm.com/docs/terraform?topic=terraform-cis-resources#cis-waf-rule)
```

Comment:
What is the difference between `WAF Package, Waf group` and WAF Rule?

Reply:
WAF Rules are in huge amount.  So, WAF Package and WAF Group are for segregate and identify what kind of WAF Rules.